packager/app/packager_main.cc

```diff
@@ -329,18 +329,6 @@ base::Optional<PackagingParams> GetPackagingParams() {
   chunking_params.segment_sap_aligned = FLAGS_segment_sap_aligned;
   chunking_params.subsegment_sap_aligned = FLAGS_fragment_sap_aligned;
 
-  if (chunking_params.is_low_latency_dash &&
```

Comment:
Isn't this change already covered in your other pull request?

Reply:
It is - I made this unit test branch off of the `restirct-low-latency-inputs` branch seen in the other PR. This difference will disappear once the `restirct-low-latency-inputs` PR is merged into the `low-latency-dash-support-staging branch`